### PR TITLE
Adds first iteration of logs2webhook service

### PR DESCRIPTION
**Pull request by bomoko**
Addresses #1811 

- [ ] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated
- [ ] PR title is ready for changelog and subsystem label(s) applied

Explain the **details** for making this change. What existing problem does the pull request solve?

We've found that the notifications system could fruitfully be used to also drive webhooks. This PR introduces a copy-pasta'ed service sends data to an arbitrarily defined webhook url.

### CODE REVIEW

**Comment by bomoko** [comment]
@tobybellwood - added you to review.

Still need to test this in context (I've been injecting events into the broker to test) - so let's do that together.

**Comment by tobybellwood** [comment]
so, to make this tie in with the service addition in lagoon-charts, it's a multi-step process:

1. Update the references in the Makefile in https://github.com/uselagoon/lagoon-charts/pull/246 to set
 `IMAGE_REGISTRY = testlagoon` and 
 `IMAGE_TAG = pr-2593` to grab the images built from this PR
2. Push the branch in https://github.com/uselagoon/lagoon-charts/pull/246 to uselagoon/lagoon-charts directly (i.e at feature/adds_logs2webhooks)
3. Reference that branch in the ./Makefile in this repo as `CHARTS_TREEISH=feature/adds_logs2webhooks`
4. Push this change up to the PR here to run it through Jenkins.
5. Work out how to revert all these changes when it comes time to merge (I think this one first???)

General notes - adding a new service isn't easy at the moment, and strongly favours pushing non-forked branches.  It also obviously gets more complicated when this branch also runs tests that require the new service!

File: Makefile [hunk 1]
```diff
@@ -954,7 +955,7 @@ GOJQ_VERSION = v0.12.3
 CHART_TESTING_VERSION = v3.4.0
 KIND_IMAGE = kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6
 TESTS = [api,features-kubernetes,features-kubernetes-2,nginx,drupal-php73,drupal-php74,drupal-postgres,python,gitlab,github,bitbucket,node-mongodb,elasticsearch]
-CHARTS_TREEISH = main
+CHARTS_TREEISH = feature/adds_logs2webhooks #main
```
Comment (tobybellwood) [inline Makefile]:
```suggestion
CHARTS_TREEISH = feature/adds_logs2webhooks
```

File: services/logs2webhook/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
+ARG LAGOON_GIT_BRANCH
+ARG IMAGE_REPO
+ARG UPSTREAM_REPO
+ARG UPSTREAM_TAG
+# STAGE 1: Loading Image lagoon-node-packages-builder which contains node packages shared by all Node Services
+FROM ${IMAGE_REPO:-lagoon}/yarn-workspace-builder as yarn-workspace-builder
+
+# STAGE 2: specific service Image
+FROM ${UPSTREAM_REPO:-uselagoon}/node-10:${UPSTREAM_TAG:-latest}
```
Comment (tobybellwood) [inline services/logs2webhook/Dockerfile]:
```suggestion
FROM ${UPSTREAM_REPO:-uselagoon}/node-16:${UPSTREAM_TAG:-latest}
```
node-16 as per the others

File: services/logs2webhook/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+* API [***dependency***]
+* RabbitMQ [***dependency***]
+
+## Message Queues
+
+* Consumes: `lagoon-logs`, `lagoon-logs:webhook`
+* Produces: `lagoon-logs:webhook`
+
+[documentation]: https://lagoon.readthedocs.io/
```
Comment (tobybellwood) [inline services/logs2webhook/README.md]:
```suggestion
[documentation]: https://docs.lagoon.sh/
```
docs.lagoon.sh as per the others

File: Makefile [hunk 4]
```diff
@@ -954,7 +955,7 @@ GOJQ_VERSION = v0.12.3
 CHART_TESTING_VERSION = v3.4.0
 KIND_IMAGE = kindest/node:v1.21.1@sha256:69860bda5563ac81e3c0057d654b5253219618a22ec3a346306239bba8cfa1a6
 TESTS = [api,features-kubernetes,features-kubernetes-2,nginx,drupal-php73,drupal-php74,drupal-postgres,python,gitlab,github,bitbucket,node-mongodb,elasticsearch]
-CHARTS_TREEISH = main
+CHARTS_TREEISH = feature/adds_logs2webhooks
```
Comment (tobybellwood) [inline Makefile]:
```suggestion
CHARTS_TREEISH = main
```
This is the last thing to reset (back to `main`) before merging this PR

**Pull request merged**